lib/rubocop/ast/processed_source.rb

```diff
@@ -35,25 +35,21 @@ class ProcessedSource # rubocop:disable Metrics/ClassLength
       INVALID_LEVELS = %i[error fatal].freeze
       private_constant :INVALID_LEVELS
 
-      PARSER_ENGINES = %i[parser_whitequark parser_prism].freeze
+      PARSER_ENGINES = %i[automatic parser_whitequark parser_prism].freeze
```

Comment:
The term "automatic" might not be the most suitable. "default" might be more preferable.

```suggestion
      PARSER_ENGINES = %i[default parser_whitequark parser_prism].freeze
```

Reply:
I'm not sure if `default` is actually better but I also don't like `automatic` particularly.

In the original version, I opted to simply use `nil` to signify this instead. But that has backwards compatibility concerns, where it might choose prism when it previously wouldn't when the user doesn't pass `parser_engine` to the method at all.

I changed it to `:default`.

Reply:
Yeah, I'm not sure if `default` is the most suitable choice, but it's a simple term. It's understood that this is a parameter for compatibility and it would be nice if a better name were suggested :-)

Reply:
Yup, most users won't have to configure this. So I wouldn't think _too_ much about it.

Reply:
Agreed.